### DISCUSSION TITLE
Similar to #431, trying to install using docker-compose - getting curl 403 error

### DISCUSSION THREAD

**Discussion by pryce-kris** [mediagis/nominatim-docker#450] [Q&A] [answered]
Similar to https://github.com/mediagis/nominatim-docker/discussions/431[https://github.com/mediagis/nominatim-docker/discussions/431](https://github.com/mediagis/nominatim-docker/discussions/431) 

I am not sure if this needs to be reopened. I am failing, even with user agent set:

curl -L -A mediagis/nominatim-docker:4.2.3 --fail-with-body https://nominatim.org/data/wikimedia-importance.sql.gz -o /nominatim/wikimedia-importance.sql.gz

Sorry if this is in the wrong place

**Comment by mtmail** [comment, marked as answer]
There's no need to link to every other github issue.

All default user agents are blocked. Set the user agent to something that identifies you, e.g. pryce-kris-at-gmail-dot-com.

User in issue 413 wrote " I've used a chrome agent and" That's also a default user agent to avoid.

**Reply by mtmail** [reply]
If the error message is "Warning: Failed to open the file /nominatim/wikimedia-importance.sql.gz" that just means the `/nominatim` path locally doesn't exist yet.

**Reply by pryce-kris** [reply]
Thank you
